cmd/acra-keys/keys/keystore.go

```diff
@@ -245,3 +245,27 @@ func openKeyStoreV2(params KeyStoreParameters, loader keyloader.MasterKeyLoader)
 	}
 	return keystoreV2.NewServerKeyStore(keyDirectory), nil
 }
+
+// IsKeyDirectory checks if the directory contains a keystore version 2 from KeyStoreParameters
+func IsKeyDirectory(params KeyStoreParameters) bool {
```

Comment:
what about to change this legacy naming from IsKeyDirectory to `CheckV2KeyStore` or `IsKeyStoreV2`?
because this function checks is it keystore v2 and doesn't check is it keystore over some folder or FileSystem.

Reply:
Agree, will rename it